packages/compass/src/main/menu.ts

```diff
@@ -455,7 +461,7 @@ class CompassMenu {
       'window:hide-collection-submenu': this.hideCollection.bind(this),
     });
 
-    preferences.onPreferenceValueChanged('theme', (newTheme) => {
+    preferences.onPreferenceValueChanged('theme', (newTheme: THEMES) => {
```

Comment:
Also just out of curiosity, not a request to change anything, was something complaining about this? Typescript should be smart enough to infer the type of `newTheme`, because `onPreferenceValueChanged` is defined as a template method

Reply:
TS was complaining that newTheme is implicitly any type. But it was before I restarted the computer on Friday, and now lots of weird stuff is gone.